### DISCUSSION TITLE
Is there any plans for building a docker container for this project?

### DISCUSSION THREAD

**Discussion by Waldorf3** [fuatakgun/eufy_security#367] [Ideas]
Not just for eufy-security-ws but the whole package,  

Sorry if it's a dumb question. I would do it myself if I had the skills.

**Comment by fuatakgun** [comment]
Whole package, you mean rtsp server + eufy?

**Comment by Waldorf3** [comment]
I suppose. Whatever is needed for me to record the feed of my eufy cam's.

**Reply by fuatakgun** [reply]
No, I dont have plan as it would require more things to take care, feel free to propose
